asset/sass/_elements.scss

```diff
@@ -30,3 +30,18 @@ pre[class*="language-"] {
     margin-top: $padding-large-vertical * 2;
     margin-bottom: $padding-large-vertical * 2;
 }
+
+.dropdown_packagename {
+  font-size: .8em;
```

Comment:
And we have also a Sass variable for the font size: `$font-size-small` 😉 

https://github.com/zendframework/zf-mkdoc-theme/blob/master/asset/sass/_settings.scss#L17

Reply:
Done.